src/Resources/ResourceManager/Implementation/Resource/GetAzureResourceCmdlet.cs

```diff
@@ -226,6 +226,9 @@ private bool ShouldConstructResourceId(out string resourceId)
                                             ResourceGroupName,
                                             ResourceType,
                                             Name);
+
+                this.DefaultApiVersion = DetermineApiVersion(resourceId).Result;
```

Comment:
Hey, can you add a test case for this?

Reply:
Any please resolve the conflicts. Thanks

Reply:
will do